ui/app/i18n-provider.js

```diff
@@ -13,6 +13,9 @@ class I18nProvider extends Component {
       t (key, ...args) {
         return t(current, key, ...args) || t(en, key, ...args) || `[${key}]`
       },
+      tOrDefault (key, ...args) {
```

Comment:
This could take a default argument e.g. `tOrDefault(key, default, ...args)` (or even just `tOrDefault(key, default)`) and then clients could pass in whatever default they want